### PR TITLE
Remove inactives from buttons dashboard (CU-fwpya5)

### PR DESCRIPTION
**Pull request by risatronic**
added is_active column to db to use for conditional display of installations in dashboard

### CODE REVIEW

**Review by trisapeace — APPROVED** [review]
This looks great! Good job. Your first Buttons pull request :tada: 

A few of housekeeping items:
* Could you please add a line in the `CHANGELOG.md` file with a short description and the ClickUp ID?
* Could you please add the ClickUp ID to the name of this pull request? This will automatically create a link to ClickUp in the pull request comments and will create some links to GitHub from ClickUp
* Feel free to add everyone as reviewers to your pull requests to be sure that we're sent email notifications about them
* Please wait for at least two people to approve your pull request before you merge
* When you merge, we often do "Squash and Merge" which will put all your changes into a single commit and you can write an appropriate comment for that one commit. This means that the `master` branch will be more understandable. But this is sort of case-by-case because sometimes it's useful to keep some commits separate with their individual comments. You'll get a feel for it.

Oh, and in general, I'm often quite chatty in pull requests. I looooove this video that talks about code review culture and how great it can be: https://www.youtube.com/watch?v=PJjmw9TRB7s

**Comment by trisapeace** [comment]
Task linked: [CU-fwpya5 Remove OPS from Buttons Dashboard](https://app.clickup.com/t/fwpya5)

File: server/server.js
Comment (mariocimet) [inline server/server.js:265]: nice, higher order functions are neat

Comment (risatronic) [reply server/server.js:265]: Thanks! 😊

**Review by mariocimet — APPROVED** [review]

**Pull request merged**
